build/Maven/settings.xml

```diff
@@ -14,9 +14,9 @@
     </server>
     <!-- Sonatype deployment -->
     <server>
-      <id>ossh</id>
+      <id>osshr</id>
       <username>claudiow</username>
-      <password>{7LQhdQy5LeQHMBP60Acv7Oa2GV0O0Yf3s6Nyk9T9YZQ=}</password>
+      <password>{H879k2Nwhr8Hd1UkT8jcmy5lboFLhKkVx2l2BAvZA/0=}</password>

```

Comment:
Yes it is supposed to be checked in. This are my git and Maven passwords.

Once the testing is done I will change my passwords, change the files and encrypt the files like I did with the certificates.

BTW I'm creating a user in Sonatype Jira called Loj4JNACI this is what we will be using for CI. 

You should create an user at Sonatype [https://issues.sonatype.org/secure/Dashboard.jspa](https://issues.sonatype.org/secure/Dashboard.jspa) in case I'm not available.

I will create a ticket to give you administrative access to `org.dblock` repositories